python/paddle/v2/reader/creator.py

```diff
@@ -55,3 +55,22 @@ def reader():
         f.close()
 
     return reader
+
+
+def RecordIO(path):
```

Comment:
We are following PEP 8 naming convention, please see: https://stackoverflow.com/a/159745/852385